packages/db/src/meta/pouch/adapters/indexeddb.ts

```diff
@@ -16,21 +12,21 @@ export interface DatabasesSettings {
 }
 
 export const getDefaultSettings: GetDefaultSettings = () => ({
-  directory: path.join(Config.getTruffleDataDirectory(), ".db", "sqlite")
+  directory: path.join(Config.getTruffleDataDirectory(), ".db")
 });
 
 export class Databases<C extends Collections> extends Base.Databases<C> {
   private directory: string;
 
   setup(settings: DatabasesSettings) {
-    this.directory = settings.directory;
+    // ensure db files reside in a path that ends with indexeddb
+    // whether specified in config, or using default
+    this.directory = path.join(settings.directory, "indexeddb");
```

Comment:
why did adding this suffix get moved? is it just more proper like this since default settings should probably be a little more like, "general"?

Reply:
This is to ensure the new truffle db persistence location doesn't conflict with existing (default and customized) locations.